content/en/blog/_posts/2022-08-23-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,177 @@
+
+
+### Promoted CSI Ephemeral Volume to Stable
+
+The [CSI Ephemeral Volume](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/596-csi-inline-volumes) feature allows CSI volumes to be specified directly in the pod specification for ephemeral use cases. They can be used to inject arbitrary states, such as configuration, secrets, identity, variables or similar information, directly inside pods using a mounted volume. This was initially introduced in 1.15 as an alpha feature, and it moved to GA. This feature is used by some CSI drivers such as the [secret-store CSI driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver).
+
+### Promoted CRD Validation Expression Language to Beta
+
+[CRD Validation Expression Language](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2876-crd-validation-expression-language/README.mdv) is promoted to beta, which makes it possible to declare how custom resources are validated using the [Common Expression Language (CEL)](https://github.com/google/cel-spec). Please see the [validation rules](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules) guide.
+
+### Promoted Server Side Unknown Field Validation to Beta
+
+Promoted the `ServerSideFieldValidation` feature gate to beta (on by default). This allows optionally triggering schema validation on the API server that errors when unknown fields are detected. This allows the removal of client-side validation from kubectl while maintaining the same core functionality of erroring out on requests that contain unknown or invalid fields.
+
+###  Introduced KMS v2 API
+
+Introduce KMS v2alpha1 API to add performance, rotation, and observability improvements. Encrypt data at rest (ie Kubernetes `Secrets`) with DEK using AES-GCM instead of AES-CBC for kms data encryption. No user action is required. Reads with AES-GCM and AES-CBC will continue to be allowed. See the guide [Using a KMS provider for data encryption](https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/) for more information.
+
+### Removed AKS and GKE authentication from kubectl
+
+The `gcp` and `azure` auth plugins have been removed from client-go and kubectl. See [AKS](https://github.com/Azure/kubelogin) and [GKE](https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke) documentation for details about the cloud-specific replacements.
```

Comment:
this removal was reverted at the last minute due to a kubectl bug and retargeted for 1.26 ... details in https://github.com/kubernetes/kubernetes/issues/111911#issuecomment-1219783870